### ISSUE TITLE
Add good styling to the checkboxes

### ISSUE THREAD

**Issue by Jittojoyes98** [Jittojoyes98/Todo-App#1]
Since its hard to style a checkbox in CSS, would like to do the same with any framework like bootstrap without loosing the functionality of the App.

**Comment by saraswatkarthik** [comment]
@Jittojoyes98

**Comment by Jittojoyes98** [comment]
Yeah, @saraswatkarthik  if you are ready to contribute give it a shot.

**Comment by saraswatkarthik** [comment]
> Yeah, @saraswatkarthik if you are ready to contribute give it a shot.

Can you give reference of code line?

**Comment by Jittojoyes98** [comment]
@saraswatkarthik  , Iam sorry that i somehow missed the styling part in the project.I will be working on it(Since i didn't deploy i actually didn't thought about the bug).I will let you know when i'am completed my task by tagging you here.
